app/imports/ui/pages/ListBills.jsx

```diff
@@ -0,0 +1,190 @@
+import React from 'react';
+import { Container } from 'react-bootstrap';
+import { MDBBadge, MDBTable, MDBTableHead, MDBTableBody, MDBPagination, MDBPaginationLink, MDBPaginationItem } from 'mdb-react-ui-kit';
+//import { PAGE_IDS } from '../utilities/PageIDs';
+//import { Meteor } from 'meteor/meteor';
+//import { useTracker } from 'meteor/react-meteor-data';
+//import { PAGE_IDS } from '../utilities/PageIDs';
+
+/* A simple static component to render some text for the listBills page. */
+const TestimonyList = () => (
+    <Container>
```

Comment:
Not sure if its just me but im getting ESLint error for bad indentation from line 11 - 187, found 4 indents when only wants 2